apps/apps.c

```diff
@@ -2294,29 +2294,28 @@ int app_access(const char* name, int flag)
 #ifdef _WIN32
 int app_isdir(const char *name)
 {
-    HANDLE hList;
-    WIN32_FIND_DATA FileData;
+    DWORD attr;
 # if defined(UNICODE) || defined(_UNICODE)
     size_t i, len_0 = strlen(name) + 1;
+    WCHAR tempname[MAX_PATH];
 
-    if (len_0 > OSSL_NELEM(FileData.cFileName))
+    if (len_0 > MAX_PATH)
         return -1;
 
 #  if !defined(_WIN32_WCE) || _WIN32_WCE>=101
     if (!MultiByteToWideChar
-        (CP_ACP, 0, name, len_0, FileData.cFileName, len_0))
+        (CP_ACP, 0, name, len_0, tempname, len_0))
```

Comment:
On side note formally speaking there is a logical inconsistency here. The call assumes that it takes equal number of characters, len_0, to represent name. But there is no guarantee that it's the case. However, it usually takes less WCHARs, so overall it works out...